openapi.yaml

```diff
@@ -3602,6 +3602,7 @@ components:
         * `pr-rejected`: number of rejected PRs.
         * `pr-closed`: number of closed PRs.
         * `pr-done`: number of released, closed unmerged, and force push dropped PRs.
+        * `pr-reviewed-ratio`: ratio of reviewed to closed PRs.
```

Comment:
It's important that this ratio is not bigger than 1. So we can not compute this ratio by dividing the number of PRs reviewed by the number of PRs closed, since it some cases it can be bigger than 1. For that reason I've mentioned that it is better to compute it as:
`pr-reviewed` / (`pr-reviewed` + `pr-not-reviewed`)